lib/taskana-core/src/test/resources/taskanaroles.properties

```diff
@@ -1,3 +1,3 @@
 taskana.roles.user = group1 | group2|teamlead_1 |teamlead_2 |user_1_1| user_1_1| user_1_2| user_2_1| user_2_2| max|elena|simone
-taskana.roles.Admin=teamlead_1|teamlead_2
-taskana.roles.businessadmin=max|Moritz
+taskana.roles.Admin=teamlead_1|teamlead_2|admin
```

Comment:
teamlead 1 and 2 should not be business admin in the acceptance test, since they are used in a special meaning.